lib/capybara/spec/session/node_spec.rb

```diff
@@ -100,9 +100,15 @@
     end
 
     it "should fill the field even if the caret was not at the end", requires: [:js] do
+      @session.first('//input').set('monkey')
       @session.execute_script("var el = document.getElementById('test_field'); el.focus(); el.setSelectionRange(0, 0);")
       @session.first('//input').set('')
       expect(@session.first('//input').value).to eq('')
+
+      @session.first('//input').set('monkey')
```

Comment:
This test can't go here.  The `clear: :backspace` behavior is Selenium specific so the test needs to go in `spec/shared_selenium_session.rb` with the other tests for that behavior

Reply:
okay, I'll move the test tomorrow. Appart from that, are you ok with the PR?

Reply:
Looking at it more -- it appears the issue is Chrome specific -- wondering whether it's worth fixing it only in chrome_node.rb - I am curious why it only affects Chrome though since it seems like setting the caret to the beginning should affect FF too.